packages/backend/src/services/UnfurlService/UnfurlService.ts

```diff
@@ -465,10 +465,14 @@ export class UnfurlService {
                             ? `[data-testid="visualization"]`
                             : 'body';
 
-                    const element = await page.waitForSelector(selector, {
+                    await page.waitForNetworkIdle({
```

Comment:
we already do this here... :thinking:  https://github.com/lightdash/lightdash/pull/8775/files#diff-9b86624a0467c9253f755c66928390e96818008cdd1df400c3509984ce4d1071L441 is this not redundant ?

Reply:
changed approach ^ resolving this.